public/javascripts/disemvoweling.js

```diff
@@ -0,0 +1,10 @@
+const disemvoweling = (stringToParse='') => {
```

Comment:
this solution it too complex. It also has formatting issues. Try something like:
```js
const disemvoweling = string => string.toLowerCase().replace(/[aeiou\s\d]/ig)
```